### DISCUSSION TITLE
incompatibilidades ASPNET.CORE 3 vs 8 ?

### DISCUSSION THREAD

**Discussion by InacioCarvalhoOliveira** [balta-io/forum#611] [Tira Dúvidas] [answered]
Bom dia a todos, estou fazendo o curso 
**[Legado] Criando APIs Data Driven com ASP.NET Core 3 e EF Core 3** contudo tenho em máquina a versão do dotnet na **8.0.105**(dotnet --version) teria alguma incompatibilide futura ao seguir com versões diferentes ?, digo, pois logo ao criar o programa, tive que trazer o arquivo **Startup.cs**

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @InacioCarvalhoOliveira, como vai?

O ideal é seguir a primeira vez com a mesma versão do curso, depois você atualiza.
Mas se quiser seguir com a versão 8, tem algumas diferenças mas não são muitas... só vai te dar mais trabalho pesquisando como resolver algumas coisas que mudaram.

Importante frisar que as mudanças são cobertas nos próximos cursos... então pode ficar tranquilo que você vai chegar até o .NET 8!

**Reply by InacioCarvalhoOliveira** [reply]
Obrigado Balta, valeu pelas informações, vou seguir então!!
